Automattic/wp-calypso
# Refactor the start flow for tailored flows

## Proposed Changes

This moves all the logic for page title and progress bar into the `SignupHeader` component. The page title is now set by query parameter and the variationName parameter is used to return the proper progress bar status.

This also removes the unused pre-launchpad loader.

## Testing Instructions

1. Pull branch and `yarn start`
2. Go through the tailored flows and ensure the flow title and progress bar appears on all pages.
3. Check /start/user and /start/domains to make sure there is no regression
